infra/feast-operator/Makefile

```diff
@@ -0,0 +1,227 @@
+# VERSION defines the project version for the bundle.
+# Update this value when you upgrade the version of your project.
+# To re-generate a bundle for another specific version without changing the standard setup, you can:
+# - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
+# - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
+VERSION ?= 0.37.1
```

Comment:
this should probably be picked up from somewhere...

Reply:
i added it to the existing release version scripting -
https://github.com/feast-dev/feast/blob/816d8f1c9ca796a689c341973c1d52678d419b69/infra/scripts/release/files_to_bump.txt#L13

and it is picked up during builds from the root Makefile - 
https://github.com/feast-dev/feast/blob/816d8f1c9ca796a689c341973c1d52678d419b69/Makefile#L407

let me know though if i should script something separately.